docs/Developers/NextGen_Core.rst

```diff
@@ -0,0 +1,196 @@
+NextGen-Core
```

Comment:
Copy the whole text and run a spell check ;) All typos would be instantly revealed.

recieves, machanism, betweeen, etc :)

Reply:
on it :+1:

Reply:
done. no more spelling errors

Reply:
@gitmate-bot rebase